### PR TITLE
Fixes wrong localhost port in quickstart guide

### PR DESCRIPTION
**Pull request by skvskv**
The app listens on 8080, not on 3000.

### CODE REVIEW

**Comment by skvskv** [comment]
screenshot from 2018-06-24 00-41-27
Is your CLA checker broken?

**Comment by Salaboy** [comment]
@skvskv it is working for me.. I double checked it with a different account. I can see this warning: 
"Konstantin Shpolvind seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account." It sounds like you don't have the correct email set in your git config or something like that.

File: getting-started/quickstart.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Once you have Docker up and running, follow the next steps
 `docker-compose -f infrastructure-docker-compose.yml up -d`  
 `docker-compose -f application-docker-compose.yml up -d`
 
-Then point your browser to [http://localhost:3000](http://localhost:3000).
+Then point your browser to [http://localhost:8080](http://localhost:8080).
```
Comment (Salaboy) [inline getting-started/quickstart.md]: @skvskv if you point to 8080 you will go to the gateway, if you are running the demo client UI it should run on 3000. am I wrong? Can you see the UI in 8080?

Comment (skvskv) [reply getting-started/quickstart.md]: 8080 works for me, while 3000 absolutely doesn't (that's why I sought for another probable port)

Comment (ryandawsonuk) [reply getting-started/quickstart.md]: Could you show a screenshot including the port? I have a feeling we mean different things by 'demo ui'

**Comment by Salaboy** [comment]
@skvskv we are closing this because we changed the quickstart guide. Thanks a lot for the feedback

**Pull request closed without merging**